src/commcare_cloud/commands/deploy/commcare.py

```diff
@@ -1,8 +1,6 @@
 import shlex
 from datetime import datetime, timedelta
 
-import pytz
```

Comment:
pytz is already imported in deploy/utils.py if you happen to notice that there isn't a diff adding it to the file.